bindings/pyroot/pythonizations/python/ROOT/_pythonization/_rntuple.py

```diff
@@ -78,11 +84,25 @@ def pythonize_RNTupleModel(klass):
     klass._CreateBare = klass.CreateBare
     klass.CreateBare = _RNTupleModel_CreateBare
 
+    klass.CreateBareEntry = _RNTupleModel_CreateEntry
+    klass.CreateEntry = _RNTupleModel_CreateEntry
     klass.GetDefaultEntry = _RNTupleModel_GetDefaultEntry
 
     klass.MakeField = MethodTemplateGetter(klass.MakeField, _RNTupleModel_MakeField)
 
 
+def _RNTupleReader_Open(maybe_model, *args):
+    if (
+        hasattr(type(maybe_model), "__cpp_name__")
+        and type(maybe_model).__cpp_name__ == "ROOT::Experimental::RNTupleModel"
+    ):
+        # In Python, the user cannot create REntries directly from a model, so we can safely clone it and avoid destructively passing the user argument.
+        maybe_model = maybe_model.Clone()
```

Comment:
Here we might have a free opportunity for some extra error handling in an `else` branch letting the user know that only an `RNTupleModel` is accepted as input.

Reply:
`RNTupleReader::Open` is overloaded, the first argument can also just be a string. In that case, the model is reconstructed from the on-disk header. I don't know if there is a better way to implement overloaded methods in Python...

Reply:
> I don't know if there is a better way to implement overloaded methods in Python...

The best approximation of it is https://docs.python.org/3/library/functools.html#functools.singledispatch , I don't know if it's worth using it for this particular PR.

Reply:
I also don't know how nicely this plays with cppyy. I'll leave the few extra cases as they are for the moment.